gitium/inc/class-gitium-menu.php

```diff
@@ -96,7 +96,7 @@ protected function show_disconnect_repository_button() {
 			<?php
 				wp_nonce_field( 'gitium-admin' );
 		  ?>
-			<input type="submit" name="GitiumSubmitDisconnectRepository" value='<?php _e( 'Disconnect from repo', 'gitium' ); ?>' class="button secondary" onclick="return confirm('<?php _e( 'Are you sure you want to disconnect from the remote repository?', 'gitium' ); ?>')"/>&nbsp;
+			<input type="submit" name="GitiumSubmitDisconnectRepository" value='<?php echo 'Disconnect from repo'; ?>' class="button secondary" onclick="return confirm('<?php echo 'Are you sure you want to disconnect from the remote repository?'; ?>')"/>&nbsp;
```

Comment:
```suggestion
			<input type="submit" name="GitiumSubmitDisconnectRepository" value="<?php echo 'Disconnect from repo'; ?>" class="button secondary" onclick="return confirm('<?php echo 'Are you sure you want to disconnect from the remote repository?'; ?>')"/>&nbsp;
```

I think you should use double quotes here. Because outer quotes will conflict with inner quotes